public/app/core/utils/timePicker.ts

```diff
@@ -30,9 +30,11 @@ export const getShiftedTimeRange = (direction: number, origRange: TimeRange): Ab
 export const getZoomedTimeRange = (range: TimeRange, factor: number): AbsoluteTimeRange => {
   const timespan = range.to.valueOf() - range.from.valueOf();
   const center = range.to.valueOf() - timespan / 2;
+  // If the timepsan is 0, zooming out would do nothing, so we force a zoom out to 30s
+  let newTimespan = timespan === 0 ? 30000 : timespan * factor;
```

Comment:
minor nit: can this be a `const`?

Reply:
Oh yeah! I played around with different ways of writing this and then forgot to change this to a const when it wasn't changed anymore 🙃